README.md

```diff
@@ -6,20 +6,99 @@
 
 The official dojo 2 build command.
 
+*WARNING* This is beta software. It is not yet production ready, so you should use at your own risk.
+
+- [Usage](#usage)
+- [Features](#features)
+  - [Building](#building)
+  - [Building a custom element](#building-a-custom-element)
+  - [Eject](#eject)
+- [How to I contribute?](#how-do-i-contribute)
+  - [Installation](#installation)
+  - [Testing](#testing)
+- [Licensing information](#licensing-information)
+
+## Usage
+
+To use `@dojo/cli-build` in a single project, install the package:
+
+```bash
+npm install @dojo/cli-build
+```
+
+to use `@dojo/cli-build` in every project, install the project globally:
+
+```bash
+npm install -g @dojo/cli-build
+```
+
 ## Features
 
-TODO: Add sections on features of this package
+`@dojo/cli-build` is an optional command for the [Dojo CLI](https://github.com/dojo/cli).
+
+### Building
+
+To build a Dojo 2 application for publishing:
+
+```bash
+dojo build
+```
+
+This command will output the built files to the `dist` directory.  After running this command, you can open the `dist/index.html` file to see your application.
+
+You can also build in watch mode, which will automatically rebuild your application when it changes:
+
+```bash
+dojo build -w
+```
+
+`@dojo/cli-build-webpack` can be customized further. Use the help option to see everything you can do:
 
-## How do I use this package?
+```bash
+dojo build --help
+```
 
-TODO: Add appropriate usage and instruction guidelines
+### Building a custom element
+
+`@dojo/cli-build-webpack` can also build custom web elements. Custom elements are built by providing the name of a [custom element descriptor](https://github.com/dojo/widget-core#web-components).
```

Comment:
Perhaps worth mentioning that this is part of the web components spec, since we're not just building the custom element, but also the CSS and JS to support its usage as a web component via custom element?